addons/website_sale/models/website.py

```diff
@@ -516,6 +519,33 @@ def _get_product_page_grid_image_classes(self):
         return spacing_map.get(self.product_page_image_spacing) + ' ' +\
                 columns_map.get(self.product_page_grid_columns)
 
+    @api.model
+    def _send_abandoned_cart_email(self):
+        for website in self.search([]):
+            if not website.send_abandoned_cart_email:
+                continue
+            SaleOrder = self.env['sale.order']
+            abandoned_carts = SaleOrder.search([
+                ('is_abandoned_cart', '=', True),
+                ('cart_recovery_email_sent', '=', False)
```

Comment:
Also something we do is always end the line with a , (comma) so that there is minimal diff if we need to add a line :)